Documentation/guides/OneDotNet.md

```diff
@@ -189,12 +189,45 @@ It is recommended to migrate to the new linker settings, as
 There are currently a few "verbs" we are aiming to get working in
 Xamarin.Android:
 
+    dotnet new
     dotnet build
     dotnet publish
     dotnet run
 
-Currently in .NET 5 console apps, `dotnet publish` is where all the
-work to produce a self-contained "app" happens:
+### dotnet new
+
+To support `dotnet new`, we created a few basic project and item
+templates for Android that are named following the patterns and naming
+of existing .NET templates:
+
+    Templates                                     Short Name           Language    Tags
+    --------------------------------------------  -------------------  ----------  ----------------------
+    Android Activity template                     android-activity     [C#]        Android
+    Android Java Library Binding                  android-bindinglib   [C#]        Android
+    Android Layout template                       android-layout       [C#]        Android
+    Android Class library                         androidlib           [C#]        Android
+    Android Application                           android              [C#]        Android
+    Console Application                           console              [C#],F#,VB  Common/Console
```

Comment:
May want to omit the entries that *don't* appear to follow convention, so *skip* `console` and `classlib` & the rest, but *include* `wpf` and `wpflib`.

Reply:
I included `wpf`, `wpflib`, `nunit`, and `nunit-test`. I don't think we should drop `console` or `classlib` from this example, because those are the most basic templates.